README.md

```diff
@@ -110,3 +110,9 @@ To store a remote configuration in the environment, use:
 ```sh
 > comfy get development | source /dev/stdin
 ```
+
+To launch admin webApp on localhost:3000:
```

Comment:
We should avoir hardcoding the port here. If this app is meant to be used in development too, something might already be running on the very common 3000 port